src/Tree/Strategy/ORM/Nested.php

```diff
@@ -647,7 +647,7 @@ public function shiftRangeRL(EntityManagerInterface $em, $class, $first, $last,
         $sign = ($delta >= 0) ? ' + ' : ' - ';
         $absDelta = abs($delta);
         $levelSign = ($levelDelta >= 0) ? ' + ' : ' - ';
-        $absLevelDelta = abs($levelDelta);
+        $absLevelDelta = $levelDelta ? abs($levelDelta) : 0;
```

Comment:
```suggestion
        $absLevelDelta = null !== $levelDelta ? abs($levelDelta) : 0;
```
To use a stricter comparison